src/sources/file.rs

```diff
@@ -518,10 +515,11 @@ mod tests {
 
         let dir = tempdir().unwrap();
         let config = file::FileConfig {
-            include: vec![dir.path().join("*")],
+            include: vec![dir.path().join("*").to_str().unwrap().to_owned()],
```

Comment:
This is a frequent enough pattern you might want a convenience function for it.

Reply:
Considering this is only used for test assertions, I feel abstracting over this is not worth it.